api/incubator/src/main/java/io/opentelemetry/api/incubator/events/EventLogger.java

```diff
@@ -32,18 +31,6 @@
 @ThreadSafe
 public interface EventLogger {
 
-  /**
-   * Emit an event.
-   *
-   * @param eventName the event name, which identifies the class or type of event. Event with the
-   *     same name are structurally similar to one another. Event names are subject to the same
-   *     naming rules as attribute names. Notably, they are namespaced to avoid collisions. See <a
-   *     href="https://opentelemetry.io/docs/specs/semconv/general/events/">event.name semantic
-   *     conventions</a> for more details.
-   * @param attributes attributes associated with the event
-   */
-  void emit(String eventName, Attributes attributes);
```

Comment:
Is the intent to force the user to use the builder? Are we just keeping it simple for now and adding as needed later?

Reply:
Yes, only way to do it now is with the builder. I'm open to re-adding this type of syntactic sugar method, but want to understand more about event body fields interact with attributes first, since I think the answer that impacts the API design. Need to see how the decision https://github.com/open-telemetry/semantic-conventions/issues/505 shakes out in `semantic-conventions` and the spec.